parkhangs-expressapp/controllers/eventCommentController.js

```diff
@@ -27,16 +27,14 @@ const getEventComments = async (req, res) => {
 const addEventComment = async (req, res) => {
     var {
         eventId,
-        eventCommentId,
         comment
     } = req.body
     var newEventComment = {
         eventId: eventId ? eventId : null,
-        eventCommentId: eventCommentId ? eventCommentId : null,
         comment: comment ? comment : null
     }
 
-    if (newEventComment.eventId === null || newEventComment.eventCommentId === null || newEventComment.comment === null) {
+    if (newEventComment.eventId === null || newEventComment.comment === null) {
         return res.status(400).json({success: false, error: `Missing one or more fields`})
     }
```

Comment:
couldn't see it on this page but:  https://github.com/cmaija/ParkHangs/blob/040360c30bee07395145ca8f3eea7b68a6e5e6ae/parkhangs-expressapp/controllers/eventCommentController.js#L42

we will need to pass in the user object here, okay for now